workflow/common/convert.go

```diff
@@ -7,26 +7,33 @@ import (
 	wfv1 "github.com/argoproj/argo/pkg/apis/workflow/v1alpha1"
 )
 
-func ConvertCronWorkflowToWorkflow(cronWf *wfv1.CronWorkflow) (*wfv1.Workflow, error) {
```

Comment:
@simster7 could you give your opinion on this change to implementation - should we copy labels and annotations from the cron to the created workflow or not?

Reply:
I'm thinking we should not copy all labels and annotations blindly, but instead explicitly

Reply:
@sarabala1979 @jessesuen - thoughts?

Reply:
I am agreed with @simster7. We should not copy all labels from the owner Objects. It might be specific to that `Kind`. We can pick and choose if we really need any information from Owner Object.